app/models/item.rb

```diff
@@ -34,7 +34,7 @@ class Item < ApplicationRecord
   belongs_to :kit, optional: true
   belongs_to :item_category, optional: true
 
-  validates :name, uniqueness: { scope: :organization }
+  validates :name, uniqueness: { scope: :organization, case_sensitive: false, message: "- An item with that name already exists (could be an inactive item)." }
```

Comment:
The teeniest, tiniest nit pick -- let's remove the period after (could be an inactive item)    
Why -- if you have several errors, you may end up with a period then a comma, as in this example:  "Base item must exist, Name - An item with that name already exists (could be an inactive item)., and Distribution quantity must be greater than 0"    Not having the period at the very end is better than having that ".,"     Thank you.

Reply:
Gotcha, that makes sense! Removed the period and pushed up the changes 👍🏽